xml/System.Web.UI.WebControls/LinqDataSourceView.xml

```diff
@@ -1821,9 +1932,17 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the name or names of the properties to use during the select operation for sorting data.</summary>
+        <value>A value that is used to create the OrderBy clause.</value>
```

Comment:
I think "OrderBy" needs code brackets to prevent translation.

Reply:
Same as previous comment. This is ECMAXML.

Reply:
doesn't ECMAXML support `<code>GroupBy</code>` ? I can use it in visual studio /// comments.

Reply:
Oh yeah. We can use <c> as well I believe. Let me check.